Fix build on Windows

1.
```
lib_base.lib(base_system_media_controls_win.obj) : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
lib_base.lib(base_system_unlock_win.obj) : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
lib_ui.lib(ui_windows_direct_manipulation.obj) : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
lib_webview.lib(webview_windows_edge_html.obj) : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
windows_autostart_task.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
current_geo_location_win.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
webrtc_environment_win.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
webrtc_loopback_adm_win.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
integration_win.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
notifications_manager_win.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
specific_win.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
windows_app_user_model_id.obj : error LNK2001: unresolved external symbol WINRT_IMPL_RoOriginateLanguageException
```

Should link ```runtimeobject``` in ```cmake\options_win.cmake```

2.
Remove ```/DELAYLOAD:avrt.dll``` in ```Telegram\CMakeLists.txt```

```
LINK : warning LNK4199: /DELAYLOAD:avrt.dll ignored; no imports found from avrt.dll
```

--- a/.github/workflows/win.yml
+++ b/.github/workflows/win.yml
@@ -4,6 +4,7 @@ on:
   push:
     paths-ignore:
       - 'docs/**'
+      - '!docs/building-win*.md'
       - '**.md'
       - 'changelog.txt'
       - 'LEGAL'
@@ -22,6 +23,7 @@ on:
   pull_request:
     paths-ignore:
       - 'docs/**'
+      - '!docs/building-win*.md'
       - '**.md'
       - 'changelog.txt'
       - 'LEGAL'
@@ -69,25 +71,33 @@ jobs:
         shell: bash
         run: echo "REPO_NAME=${GITHUB_REPOSITORY##*/}" >> $GITHUB_ENV
 
-      - uses: ilammy/msvc-dev-cmd@v1.13.0
-        name: Native Tools Command Prompt.
-        with:
-          arch: ${{ matrix.arch }}
-
       - name: Clone.
         uses: actions/checkout@v4
         with:
           submodules: recursive
           path: ${{ env.TBUILD }}\${{ env.REPO_NAME }}
 
-      - name: Set up environment paths.
+      - name: First set up.
         shell: bash
         run: |
-          echo "CACHE_KEY=$(sha256sum $TBUILD/$REPO_NAME/$PREPARE_PATH | awk '{ print $1 }')" >> $GITHUB_ENV
+          DOCPATH=$TBUILD/$REPO_NAME/docs/building-win.md
+          [ "${{ matrix.arch }}" != Win32 ] && DOCPATH=$TBUILD/$REPO_NAME/docs/building-win-${{ matrix.arch }}.md
+          SDK="$(grep "SDK version" $DOCPATH | sed -r 's/.*\*\*(.*)\*\* SDK version.*/\1/')"
+          echo "SDK=$SDK" >> $GITHUB_ENV
+
+          echo "$(sha256sum $TBUILD/$REPO_NAME/$PREPARE_PATH | awk '{ print $1 }')" >> CACHE_KEY.txt
+          echo "$SDK" >> CACHE_KEY.txt
+          echo "CACHE_KEY=$(sha256sum CACHE_KEY.txt | awk '{ print $1 }')" >> $GITHUB_ENV
 
           echo "Configurate git for cherry-picks."
           git config --global user.email "you@example.com"
           git config --global user.name "Sample"
+
+      - uses: ilammy/msvc-dev-cmd@v1.13.0
+        name: Native Tools Command Prompt.
+        with:
+          arch: ${{ matrix.arch }}
+          sdk: ${{ env.SDK }}
 
       - name: NuGet sources.
         run: |

